### ISSUE TITLE
Merge All Window Feature (workspace tabs)

### ISSUE THREAD

**Issue by mertcanaltin** [zed-industries/zed#7136]
### Check for existing issues

- [ ] Completed

### Describe the feature

greetings, in zed  would be great to have the merge all window feature in vscode, developers working on multiple projects can manage 3 or more project screens on a single screen with this feature.

### If applicable, add mockups / screenshots to help present your vision of the feature

<img width="1728" alt="Screenshot 2024-01-31 at 10 22 19" src="https://github.com/zed-industries/zed/assets/37827216/b4a0b18d-c3ed-490a-a7a8-44ea35558480">

**Comment by MikeSpock** [comment]
👆 This one would be a big win. JetBrains added this into its IDEs quite recently, and it is one of the most essential tools I use.

**Comment by sam0x17** [comment]
Yeah this would be really useful. In my case I am quite used to using vscode's merge window just to find all the merge conflicts easily and then clicking "Accept Incoming" / "Accept Both" etc as needed

**Comment by MikeSpock** [comment]
@sam0x17 I believe this feature idea is about merging project windows into a single window's tabs, and not about GIT merge issue handling.

**Comment by ajdorexyz** [comment]
+1 for this, I use the native Mac OS tabs a lot in VScode. I'm really enjoying Zed on the whole though ❤️

**Comment by notpeter** [comment]
I'm going through some old issues and I believe this is effectively the same as:
- https://github.com/zed-industries/zed/issues/4948

Or more specifically, if we implemented tabbed workspaces 'Merge All Windows' would very likely be part of that implementation.  Thanks for reporting.

**Comment by dalisoft** [comment]
I would love to see this feature. Seems same issue as #4948
